generator/src/main/java/org/corfudb/generator/operations/SleepOperation.java

```diff
@@ -0,0 +1,29 @@
+package org.corfudb.generator.operations;
+
+import java.util.Random;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.generator.State;
+
+/**
+ * Created by maithem on 7/14/17.
+ */
+@Slf4j
+public class SleepOperation extends Operation {
+
+    public SleepOperation(State state) {
+        super(state);
+    }
+
+    @Override
+    public void execute() {
+        Random rand = new Random();
+
+        int  sleepTime = rand.nextInt(50);
+        try {
+            Thread.sleep(sleepTime);
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)